shell/gpu/gpu_surface_metal.mm

```diff
@@ -15,25 +17,26 @@
 
 namespace flutter {
 
-GPUSurfaceMetal::GPUSurfaceMetal(fml::scoped_nsobject<CAMetalLayer> layer,
-                                 sk_sp<GrDirectContext> context,
-                                 fml::scoped_nsprotocol<id<MTLCommandQueue>> command_queue)
-    : layer_(std::move(layer)),
-      context_(std::move(context)),
-      command_queue_(std::move(command_queue)) {
-  layer_.get().pixelFormat = MTLPixelFormatBGRA8Unorm;
-  // Flutter needs to read from the color attachment in cases where there are effects such as
-  // backdrop filters.
-  layer_.get().framebufferOnly = NO;
+namespace {
+struct MTLDrawableDeleter {
```

Comment:
This should not be necessary as there is fml::CFRef for CoreFoundation types and fml::scoped_nsobject for Objective-C types already.

Reply:
Done!